src/control/cmd/dmg/pool_test.go

```diff
@@ -494,7 +493,7 @@ func TestPoolGetACLToFile_Success(t *testing.T) {
 
 	conn := newTestConn(t)
 	err := runCmd(t,
-		fmt.Sprintf("pool get-acl --pool 12345678-1234-1234-123456789abc --outfile %s", aclFile),
+		fmt.Sprintf("pool get-acl --pool 031bcaf8-f0f5-42ef-b3c5-ee048676dceb --outfile %s", aclFile),
```

Comment:
Any reason for changing this to a more plausibly realistic UUID? I think it doesn't actually matter.

Reply:
Yeah, it was failing validation, so I made it a valid UUID.